integration-cli/docker_cli_volume_test.go

```diff
@@ -20,6 +22,18 @@ func (s *DockerSuite) TestVolumeCliCreate(c *check.C) {
 	c.Assert(name, check.Equals, "test")
 }
 
+func (s *DockerSuite) TestVolumeCliCreateOptionConflict(c *check.C) {
+	dockerCmd(c, "volume", "create", "--name=test")
+	out, _, err := dockerCmdWithError("volume", "create", "--name", "test", "--driver", "nosuchdriver")
+	c.Assert(err, check.NotNil, check.Commentf("volume create excepted to be failed on creating an existed volume with a different driver name"))
```

Comment:
c.Assert(err, check.NotNil, check.Commentf("volume create excepted to be failed on creating an existed volume with a different driver name"))

c.Assert(err, check.NotNil, check.Commentf("volume create exception volume name already in use with another driver"))